Comp2022/src/main/cpp/subsystems/Vision.cpp

```diff
@@ -28,20 +31,40 @@ Vision::Vision()
     // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
 
     table = nt::NetworkTableInstance::GetDefault().GetTable("limelight");
-    table->PutNumber("ledMode", LED_CUR_MODE);
+    table->PutNumber("ledMode", LED_ON);
     table->PutNumber("stream", PIP_SECONDARY);
 
     frc::SmartDashboard::SetDefaultBoolean("VI_SM_OVERRIDE_ENABLED", false);
 
     Initialize();
 }
 
+double Vision::CalculateDist()
+{
+    frc2135::RobotConfig *config = frc2135::RobotConfig::GetInstance();
+    config->GetValueAsDouble("DTL_Distance1", m_distance1, 0.0);
+    config->GetValueAsDouble("DTL_Distance2", m_distance2, 0.0);
+    config->GetValueAsDouble("DTL_VertOffset1", m_vertOffset1, 0.0);
+    config->GetValueAsDouble("DTL_VertOffset2", m_vertOffset2, 0.0);
+
+    /*m_distance1 = frc::SmartDashboard::GetNumber("DTL_Distance1", m_distance1);

```

Comment:
wouldn't having these values on the dashboard makes it easier to calibrate when testing? or was it not needed?

Reply:
It would be easier to calibrate in the dashboard so I just added a method to get the values used to calculate the slope